src/fu-engine.c

```diff
@@ -7698,6 +7720,17 @@ fu_engine_load(FuEngine *self, FuEngineLoadFlags flags, FuProgress *progress, GE
 	}
 	fu_progress_step_done(progress);
 
+	/* migrate per-plugin settings into daemon.conf */
+	plugin_uefi = fu_plugin_list_find_by_name(self->plugin_list, "uefi_capsule", NULL);
+	if (plugin_uefi != NULL) {
+		const gchar *tmp = fu_plugin_get_config_value(plugin_uefi, "OverrideESPMountPoint");
+		if (tmp != NULL && g_strcmp0(tmp, fu_config_get_esp_location(self->config)) != 0) {
+			g_debug("migrating OverrideESPMountPoint=%s to EspLocation", tmp);
```

Comment:
I think we should probably change the original config too, right?

Reply:
Also, I think this message should _not_ be debug.  It should be an information level, we want people to know it happened in their system journal.

Reply:
Agreed, fixed.